bin/templates/project/build.gradle

```diff
@@ -197,7 +197,12 @@ android {
                 }
             }
         }
-    } else if (!cdvVersionCode) {
+    }
+    /*
+
+    ELSE NOTHING! DON'T MESS WITH THE VERSION CODE IF YOU DON'T HAVE TO!
```

Comment:
If you're building multiple apks for different architectures (for Crosswalk) - there is a need to generate unique version codes.